### PR TITLE
[Release 3.0.1] Charts and Docs updated

### PR DESCRIPTION
**Pull request by adityagajbhiye**
- Charts update with default values.
- Updated the README.md .

### CODE REVIEW

File: InstallationGuide.md [hunk 1]
```diff
@@ -1,50 +1,39 @@
 # Installation Guide
-## Product DFT
-Install from the command line:
 
-docker container run -d --name [conatainer_name] ghcr.io/catenax-ng/tx-dft-backend:[tag]
+It is necessary to inject the environment variables, credentials and URLs that can be found on application.properties file.
+please refer configuration section from here [README.md](README.md)
 
+### RUN SDE backend in ArgoCD 
+ We have helm chart available for ArgoCD deployment. In deployment, if don't specified specific version, the latest version on main is automatically picked up by ArgoCD and deployed to the environment using Helm charts.
```
Comment (Siegfriedk) [inline InstallationGuide.md:7]: Your helm chart should not necessarily just use the latest. You could easily combine the app version and chart version to always pull the version you released.

You will need to control the version anyway for proper testing

**Review by Siegfriedk — CHANGES_REQUESTED** [review]
You are joking with me right?

I will never ever review so many misstakes on such a basic level ever again for your team.

You either let someone else in your Team review it first or you ask a collegue to check your spelling etc....

**Comment by adityagajbhiye** [comment]
Apologize!! @Siegfriedk for your Inconvenience.
@sachinargade123  Please look into it .

File: README.md [hunk 2]
```diff
@@ -1,36 +1,158 @@
...
 
 It is a standalone service which can be self-hosted. 
 It enables companies to provide their data in the Catena-X network via an EDC.
-Data is uploaded via two CSV-files. The DFT registers the data in the Digital Twin Registry and makes it accessible via an EDC.
-The DFT project has two dependencies: Digital Twins and EDC.
 
-##### For installation guide: see [InstallationGuide.md](InstallationGuide.md)
+## Important !!!
+### Deployemnt of DFT backend
+The auto setup is the central service orchestration component. The auto setup hide all complex configuration stuff for you and get SDE backend as well as Frontend service deployed for you as service. The auto setup taking all deployemnt through their specific helm charts. The auto setup knows which prerequisites and which configurations are required for the components and creates them. All dependencies and any error messages are intercepted by Auto-Setup and treated correctly and meaningfully. Therefore, Auto-Setup meets your requirements exactly.
```
[the remark this answers is not in the document]
Comment (adityagajbhiye) [reply README.md:13]: Align to Auto-Setup.

[the remark this answers is not in the document]
Comment (adityagajbhiye) [reply README.md:13]: Changed to configuration properties.

File: InstallationGuide.md [hunk 3]
```diff
@@ -1,50 +1,39 @@
...
+
+ In values.yaml you can find `default` as value for all required configuration. You need to change all those values as per your need. for refernce, please refer confguration example section.
+ 
+ As part of argo CD deployment using heml chart the postgres database dependecy will get provide automatic but for EDC, DigitalTwin and Portal you need to provide valid details as per configuration requirement other wise SDE service will get started with defualt configuration but will not work as expected.
```
[the remark this answers is not in the document]
Comment (adityagajbhiye) [reply InstallationGuide.md:11]: resolved typos.

File: InstallationGuide.md (hunk 1 above)
Comment (adityagajbhiye) [reply InstallationGuide.md:7]: Re-corrected sentence.

File: InstallationGuide.md [hunk 4]
```diff
@@ -1,50 +1,39 @@
...
-Install from the command line:
 
-docker container run -d --name [conatainer_name] ghcr.io/catenax-ng/tx-dft-backend:[tag]
+It is necessary to inject the environment variables, credentials and URLs that can be found on application.properties file.
+please refer configuration section from here [README.md](README.md)
 
+### RUN SDE backend in ArgoCD 
+ We have helm chart available for ArgoCD deployment. In deployment, if don't specified specific version, the latest version on main is automatically picked up by ArgoCD and deployed to the environment using Helm charts.
+
+ In values.yaml you can find `default` as value for all required configuration. You need to change all those values as per your need. for refernce, please refer confguration example section.
```
[the remark this answers is not in the document]
Comment (adityagajbhiye) [reply InstallationGuide.md:9]: typos resolved.

File: InstallationGuide.md [hunk 5]
```diff
@@ -1,50 +1,39 @@
...
+ helm install release-name eclipse-tractusx/dft-backend
+
+### RUN SDE Backend Locally
+#### Prerequisites
+- JDK18
+- Postgres 13.2
+
+#### Steps
+1. Clone the GitHub Repository - https://github.com/eclipse-tractusx/dft-backend
+2. Get your instance of postgres running.(Create **dftdb** new database)
```
[the remark this answers is not in the document]
Comment (sachinargade123) [reply InstallationGuide.md:32]: Corrected the sentence

**Comment by sachinargade123** [comment]
@adityagajbhiye , please close this PR and create new PR for new corrections.

**Comment by adityagajbhiye** [comment]
Closing PR.

**Pull request closed without merging**